R/pkg/R/functions.R

```diff
@@ -208,6 +208,7 @@ NULL
 #' head(select(tmp, array_contains(tmp$v1, 21), size(tmp$v1)))
 #' head(select(tmp, array_max(tmp$v1), array_min(tmp$v1)))
 #' head(select(tmp, array_position(tmp$v1, 21)))
+#' head(select(tmp, array_sort(tmp$v1)))
```

Comment:
nit: don't need separate line for each example, let's merge this with array_position?